pkg/workloads/cortex/onnx_serve/api.py

```diff
@@ -162,7 +164,10 @@ def predict(app_name, api_name):
     response = {}
 
     if not util.is_dict(payload) or "samples" not in payload:
-        return prediction_failed('top level "samples" key not found in request')
+        message = 'top level "samples" key not found in request'
+        if debug:
+            message += " (payload: {})".format(util.pp_str_flat(payload))
```

Comment:
should we be truncating all of the prints? maybe 1000 characters? could be higher than 1000 too if you think more is better

Reply:
I wonder if there is a way to determine if an object is going to be too large to print? We may run into issues like we did for the spell model if the return object is too large

Reply:
i did a `time` on curls with and without `debug=true` against imagenet. 5s difference